src/EleventyFiles.js

```diff
@@ -100,10 +100,7 @@ class EleventyFiles {
 
   // TODO make this a getter
   getTemplateData() {
-    if (!this.templateData) {
-      this.templateData = new TemplateData(this.inputDir);
-    }
-    return this.templateData;
+    return this.templateData ? this.templateData : new TemplateData(this.inputDir);
```

Comment:
I'm not sure this code does the same thing.
The old code would cache the value of `new TemplateData(this.inputDir)` into `this.templateData`, whereas I don't think this code does, so it's possible `this.templateData` would never get set, always returns a falsy value to the inline `if` and we'd be calculating `new TemplateData(this.inputDir)` each time.

Reply:
Thanks @pdehaan, I missed the caching purpose of the code I was replacing!
Since the checks passed, I imagine that there's no test in place for this caching?